pkg/controller/certificates/groupapprove.go

```diff
@@ -84,3 +87,21 @@ func (cc *groupApprover) AutoApprove(csr *certificates.CertificateSigningRequest
 	})
 	return cc.client.UpdateApproval(csr)
 }
+
+func hasKubeletUsages(usages []certificates.KeyUsage) bool {
```

Comment:
I'm guessing we'll want something similar for server usages shortly. something like this might be more generally useful:
```
var clientUsages = []certificates.KeyUsage{certificates.UsageKeyEncipherment, certificates.UsageDigitalSignature, certificates.UsageClientAuth}

func usagesAreAllowed(allowedUsages, requestedUsages []certificates.KeyUsage) bool {
  ...
}
```